server/scripts/add_github_columns.py

```
@@ -0,0 +1,81 @@
+                print("Adding github_username column...")
+                conn.execute(text("""
+                    ALTER TABLE public.users 
+                    ADD COLUMN github_username VARCHAR(100)
+                """))
+                conn.commit()
+                print("✓ Added github_username column")
+            else:
+                print("✓ github_username column already exists")
+                
+        print("\n✅ Migration completed successfully!")
+        return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)